content/integrations/jira.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Datadog-Jira Integration
+integration_title: Jira
+kind: integration
+doclevel: basic
+---
+
+### Overview
+
+JIRA is an issue and project tracking system for software teams. This integration allows you to create tickets from triggered alerts in Datadog, and update existing tickets with new information as it arises. Additionally, you can see JIRA ticket creations as events within Datadog to overlay with all of your metrics.
+
+INSERT NEW TICKET EVENT PHOTO HERE
+
+
+
+### Installation
+
+Navigate to your Jira account
+
+Go to settings (Gear icon) –> Applications
+<img src="/static/images/jira/JiraInstallation2.png" style="width: 60%;display:block;"/>
+
+Under "Integrations" in left menu, Select Application Links
+<img src="/static/images/jira/JiraInstallation3.png" style="width: 40%;display:block;"/>
+
+Enter app.datadoghq.com as the URL to link –> press "Create new link"
+<img src="/static/images/jira/JiraInstallation4.png" style="width: 60%;display:block;"/>
+
+Fill in Application Name with any name (used simply for identification)
+
+Leave Generic Application Selected
+
+Check "Create Incoming Link"
+
+Press Continue
+<img src="/static/images/jira/JiraInstallation5.png" style="width: 50%;display:block;"/>
+
+Copy and Paste the Consumer Key, Consumer Name, and Public Key from the Jira tile
+
+Press Continue
+<img src="/static/images/jira/JiraInstallation6.png" style="width: 60%;display:block;"/>
+
+Navigate back to the Jira Tile
```

Comment:
anything done in the datadog app, like in the tile, or in the datadog config files is configuration. anything outside of that, like on some 3rd party host or app is installation